Assets/Mirror/Components/InterestManagement/SpatialHashing/SpatialHashingInterestManagement.cs

```diff
@@ -100,6 +103,13 @@ internal void Update()
 
                     // put into grid
                     grid.Add(position, connection);
+
```

Comment:
clientOwnedObjects includes the primary player object, so this needs to be reworked as:
```cs
if (projectNearClientOwnedObjects)
{
    foreach (NetworkIdentity clientOwnedObject in connection.clientOwnedObjects)
    {
        Vector2Int clientOwnedObjectPosition = ProjectToGrid(clientOwnedObject.transform.position);
        grid.Add(clientOwnedObjectPosition, connection);
    }
}
else
{
    Vector2Int position = ProjectToGrid(connection.identity.transform.position);
    grid.Add(position, connection);
}
```